views/index.erb

```diff
@@ -0,0 +1,23 @@
+<div class="new_post">
+  <form method="post" action="/new">
+    <textarea class="text-input" name="message" placeholder="Enter a peep (max 200 char)..."></textarea><br>
+    <input class="submit button" type="submit" value="Post">
+  </form>
+
+  <%= flash[:empty] %>
+</div>
+
+<div class="search">
+  <form method="get" action="/search">
+    <input class="search button" type="submit" value="Search">
+  </form>
+</div>
```

Comment:
Would it make more sense from a user's perspective to have the search bar on the index page that contains the peeps?